CHANGELOG.md

```diff
@@ -3,6 +3,106 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v7.0.0-beta.2
+
+_Feb 9, 2024_
+
+We'd like to offer a big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 🏃 Add slot typings on the Data Grid components (#11795) @romgrk
```

Comment:
Nitpick: I've used this emoji to signal performance improvement. 🤔 
Maybe we could use something more appropriate here?
```suggestion
- 🚀 Add slot typings on the Data Grid components (#11795) @romgrk
```